bin/control.sh

```diff
@@ -1,164 +1,104 @@
 #!/usr/bin/env bash
-if [ ! -z "${IGNITE_SCRIPT_STRICT_MODE:-}" ]
-then
+if [ ! -z "${IGNITE_SCRIPT_STRICT_MODE:-}" ]; then
     set -o nounset
     set -o errexit
     set -o pipefail
     set -o errtrace
     set -o functrace
 fi
 
-#
-# Copyright 2019 GridGain Systems, Inc. and Contributors.
-#
-# Licensed under the GridGain Community Edition License (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-#
+# Copyright and license information...
```

Comment:
@dy2285 @ibelyakov 
Could you please clarify why the license header was removed here and below [jvmdefaults.bat](https://github.com/gridgain/gridgain/pull/3241/files#diff-aafdd195855612eb2e867292ee556b070cb702c60e445710ac3fc21be9c9b704)?

Reply:
Unintended removed during a copy-paste while refactoring the script. Will restore it. Thanks for pointing it out.